framework/core/js/src/forum/components/NewAccessTokenModal.tsx

```diff
@@ -0,0 +1,67 @@
+import Modal, { IInternalModalAttrs } from '../../common/components/Modal';
+import Mithril from 'mithril';
+import Button from '../../common/components/Button';
+import app from '../app';
+import AccessToken from '../../common/models/AccessToken';
+import Stream from '../../common/utils/Stream';
+import { SaveAttributes } from '../../common/Model';
+
+export interface INewAccessTokenModalAttrs extends IInternalModalAttrs {
+  onsuccess: (token: AccessToken) => void;
+}
+
+export default class NewAccessTokenModal<CustomAttrs extends INewAccessTokenModalAttrs = INewAccessTokenModalAttrs> extends Modal<CustomAttrs> {
+  protected titleInput = Stream('');
+
+  className(): string {
+    return 'Modal--small NewAccessTokenModal';
+  }
+
+  title(): Mithril.Children {
+    return app.translator.trans('core.forum.security.new_access_token_modal.title');
+  }
+
+  content(): Mithril.Children {
+    return (
+      <div className="Modal-body">
+        <div className="Form Form--centered">
+          <div className="Form-group">
+            <input
```

Comment:
Missing label? If we want to use the modal title, we should add an `aria-labelled-by`, or we can create a visually-hidden (`.sr-only`) label element wrapping this.